classes/queue_process.php

```diff
@@ -83,125 +89,167 @@ public function create_client($handler = null) {
 
         // Only create client if it hasn't already been done.
         if (!isset($this->client)) {
-            $this->client = client_factory::get_client('\Aws\Sqs\SqsClient', $connectionoptions);
+            $this->client = new SqsClient($connectionoptions);
```

Comment:
The old factory::get_client had proxy + timeouts configured. Is that carried across?